### PR TITLE
Final

### PR DESCRIPTION
**Pull request by PramudithaWeerakoon**
laksala pull

### CODE REVIEW

**Review by PramudithaWeerakoon — COMMENTED** [review]
This code is 100% completed

File: .github/workflows/final_laksala.yml
Comment (Pramuditha2003) [inline .github/workflows/final_laksala.yml:1]: newly added

File: .github/workflows/final_nibmcourse.yml
Comment (Pramuditha2003) [inline .github/workflows/final_nibmcourse.yml:1]: newly added

File: .github/workflows/azure-static-web-apps-delightful-rock-0e1805810.yml
Comment (Pramuditha2003) [inline .github/workflows/azure-static-web-apps-delightful-rock-0e1805810.yml:1]: newly added

**Review by PramudithaWeerakoon — COMMENTED** [review]
newly added

**Review by PramudithaWeerakoon — COMMENTED** [review]
newly added

**Comment by Pramuditha2003** [comment]
newly added

**Pull request merged**